sdk/storage/Azure.Storage.Blobs.Batch/src/Models/Internal/BlobDeleteType.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Blobs.Batch.Models
+{
+    internal enum BlobDeleteType
+    {
+        /// <summary>
+        /// None.
+        /// </summary>
+        None,
+
+        /// <summary>
+        /// Permanent.
+        /// </summary>
+        Permanent
```

Comment:
was there more verbose doc for this before?

Reply:
This is an internal class that customers cannot access.